src/lib.rs

```diff
@@ -25,7 +25,6 @@
 //! assert_eq!(rx.recv().unwrap(), 42);
 //! ```
 
-#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Comment:
Why was this removed entirely? Just changing `feature(doc_cfg, doc_auto_cfg)` into `feature(doc_cfg)` should be sufficient.